lib/rsk-utils-legacy.js

```diff
@@ -2,8 +2,10 @@ const expect = require('chai').expect;
 const btcClientProvider = require('./btc-client-provider');
 const { sequentialPromise, wait, interval } = require('./utils');
 const { getBridgeAbi, getLatestActiveForkName } = require('./precompiled-abi-forks-util');
+const waitForBlockAttemptTimeMillis = process.env.WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS || 200;
+const waitForBlockMaxAttempts = process.env. WAIT_FOR_BLOCK_MAX_ATTEMPTS || 160;
```

Comment:
The tests are failing when executed against this branch. I think it might be related to this, or maybe it's the default values set in `configure_rit_locally.sh` file.

Could you set them back to the original values please?